shard-operator-parent/shard-operator-core/src/main/resources/templates/bridge-ingress-authorization-policy.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
 metadata:
   labels:
-    app.kubernetes.io/managed-by: bridge-fleet-shard-operator # used for filtering of Deployments created by the controller
-    app.kubernetes.io/created-by: bridge-fleet-shard-operator # Specify the operator
+    app.kubernetes.io/managed-by: "" # used for filtering of Deployments created by the controller
```

Comment:
Are empty strings intentional?

Reply:
@jiripetrlik Yes the empty string is intentional. Previously the name of the owning operator was hardcoded. In this PR I have updated it to be configurable meaning that two versions of the operator can create these resources from the same templates.

Much of the logic to create a Knative Broker in `v2`  is identical to `v1`, so I wanted to get as much re-use as possible.